lib/kdump_utils.pm

```diff
@@ -192,6 +192,8 @@ sub activate_kdump {
         $expect_restart_info = 1;
     }
     send_key('alt-o');
+    # Expect yast2-kdump-restart-info on s390x
+    $expect_restart_info = 1 if (is_s390x && is_sle('15-SP5+'));
```

Comment:
base https://openqa.nue.suse.com/tests/9474977#step/yast2_kdump/18 i found s390 default kdump is enable, so nothing is changed, but a kdump restart page popup after push OK button, i suppose this is an issue of s390?

Reply:
I'm not sure this is a feature or a bug, maybe we can ask a kernel expert about this or just file a bug and wait developer to confirm?

Reply:
I think this is an bug if nothing updated by yast from end user view.

Reply:
I would suggest to open a bug for s390x and use `record_soft_failure` mentioning the bug here. We can delete the `record_soft_failure` line afterwards if it is confirmed to be intended instead.

Reply:
Thanks for comments.
I got the info from https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14887#issuecomment-1124851781 that this different behavior is not a bug. And in fact I have filed a bug for s390 (https://bugzilla.suse.com/show_bug.cgi?id=1203237) but no response till now.
So I changed code to suit it for different behaviors on test side.

Reply:
That's great, thanks for the info!
All good then, I would maybe suggest to add a `record_soft_failure('bsc#1203237 - s390x openQA test fails in yast2_kdump - reboot page popup unexpected')` just to keep track of this adaptation in the code

Reply:
YaST makes some calculation and depends on that the popup appears o not:
yast2_ncurses_textmode
it depends of course of settings of the job.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14887#issuecomment-1124851781